livesync_citadel/setup.py

```diff
@@ -0,0 +1,37 @@
+# This file is part of the Indico plugins.
+# Copyright (C) 2002 - 2020 CERN
+#
+# The Indico plugins are free software; you can redistribute
+# them and/or modify them under the terms of the MIT License;
+# see the LICENSE file for more details.
+
+from __future__ import unicode_literals
+
+from setuptools import find_packages, setup
+
+
+setup(
+    name='indico-plugin-livesync-citadel',
+    version='1.0',
+    url='https://github.com/indico/indico-plugins',
+    license='MIT',
+    author='Indico Team',
+    author_email='indico-team@cern.ch',
+    packages=find_packages(),
+    zip_safe=False,
+    include_package_data=True,
+    install_requires=[
+        'indico>=2.0',
```

Comment:
`2.3.dev0` (please copy from another up-to-date plugin that's currently on a 2.3-dev version, the contents here may be a bit outdated)